### PR TITLE
onboarding experience

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

File: views/index-angular.html [hunk 1]
```diff
@@ -39,6 +39,7 @@
     <link rel="stylesheet" href="https://netdna.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.css">
     <!-- Note: this file is rendered with Less middleware can be found at /less/explore/main.less -->
     <link rel="stylesheet" ng-href="/css/angular/main.{{direction}}.css">
+    <link rel="stylesheet" href="//code.cdn.mozilla.net/fonts/fira.css">
```
Comment (secretrobotron) [inline views/index-angular.html]: I *think* this should go in `less/webfonts.less`.

File: views/layout.html [hunk 2]
```diff
@@ -31,6 +31,7 @@
 
   <title>{% block title %}{% endblock %} - {{ gettext("Mozilla Webmaker") }}</title>
   {% block css %}{% endblock %}
+  <link rel="stylesheet" href="//code.cdn.mozilla.net/fonts/fira.css">
```
Comment (secretrobotron) [inline views/layout.html]: This too

File: public/js/angular/controllers.js [hunk 3]
```diff
@@ -111,10 +111,50 @@ angular
...
 
-      $('#home-start-form').on('submit', function () {
-        $rootScope.joinWebmaker($('.home-email-field').val());
-      });
-
+      if ($routeParams.variant === "6") {
+        $('#home-start-form').on('submit', function () {
+          $rootScope.joinWebmaker($('.home-email-field').val(), "", $('#onboarding-checkbox').prop('checked'));
+        });
+        var inputText = "sharing";
```
Comment (secretrobotron) [inline public/js/angular/controllers.js]: Need this to be localizeable.

File: public/js/angular/controllers.js [hunk 4]
```diff
@@ -111,10 +111,50 @@ angular
...
+        var length = 0;
```
Comment (secretrobotron) [inline public/js/angular/controllers.js]: Could probably choose a nicer name for this var.

File: less/angular/pages/home.less [hunk 5]
```diff
@@ -307,4 +319,188 @@
...
+        margin-top: -9.8rem;
+
+        @media screen and (min-width: @screen-sm-min) {
+          margin-top: -3.8rem;
+          margin-right: 1rem;
+        }
+      }
+      .onboarding-text-content {
+        font-family: "Fira Sans";
+        font-size: 6.4rem;;
```
Comment (secretrobotron) [inline less/angular/pages/home.less]: Double semi-colon. *Super* effective.

**Comment by secretrobotron** [comment]
r-
* Fix travis!
* What is `arrow@2x.png` ? Auto-generated
* Bunch of comments.

File: less/angular/pages/home.less [hunk 6]
```diff
@@ -307,4 +319,204 @@
...
+      width: 100%;
+      @media screen and (min-width: @screen-xs-min) {
+        width: auto;
+      }
+    }
+    .onboarding-step-2 {
+      padding: 0 2rem;
+      margin-bottom: 2rem;
+      @media screen and (min-width: @screen-xs-min) {
+        padding: 0px 5rem;
```
Comment (alicoding) [inline less/angular/pages/home.less]: `padding: 0 5rem;`

File: less/angular/pages/home.less [hunk 7]
```diff
@@ -307,4 +319,204 @@
...
+        .onboarding-published-text {
+          position: absolute;
+          font-family: "Fira Sans";
+          font-size: 6.4rem;
+          font-weight: bold;
+          line-height: 1;
+          left: 22%;
+          top: 43.5%;
+          right: 10%;
+          margin: 0px;
```
Comment (alicoding) [inline less/angular/pages/home.less]: `margin: 0;`

File: public/views/home-6.html [hunk 8]
```diff
@@ -0,0 +1,155 @@
+<div ng-show="userDel" class="alert alert-info">{{ 'user_deleted' | i18n }}
+</div>
+<div id="home-splash" class="variant-6">
+  <div class="home-green home-panel">
+    <div class="container">
+      <div class="row">
+        <section class="onboarding-section">
+          <div class="onboarding-step-1">
+            <div class="onboarding-text-content">
+              <div>I <img src="/img/home/Heart.png"></div>
```
Comment (alicoding) [inline public/views/home-6.html]: I believe this needs to be localized

File: public/views/home-6.html [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+              <div class="onboarding-input-container">
+                <input type="text" class="onboarding-input"></input>
+                <div class="onboaring-input-underline"></div>
+              </div>
+            </div>
+            <div class="onboarding-tooltip-container">
+              <img class="onboarding-arrow" src="/img/home/arrow.svg">
+              <span class="onboarding-tooltip">What do you love on the web?</span>
```
Comment (alicoding) [inline public/views/home-6.html]: This.

File: public/views/home-6.html [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+            </div>
+            <button class="onboarding-next-button btn btn-default">next</button>
```
Comment (alicoding) [inline public/views/home-6.html]: And this

File: public/views/home-6.html [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+          </div>
+          <div class="onboarding-step-2 hidden">
+            <div class="onboarding-published-content">
+              <img src="/img/home/HeartPreview@2x.png"/>
+              <div class="onboarding-published-text"></div>
+            </div>
+            <div class="onboarding-left-panel">
+              <span class="hidden">
+                <h2>Good start!</h2>
```
Comment (alicoding) [inline public/views/home-6.html]: The rest of this file I guess

**Comment by alicoding** [comment]
Left some comments and most of them are minors

File: less/angular/pages/home.less (hunk 6 above)
Comment (ScottDowne) [inline less/angular/pages/home.less]: This happens when I copy it from the browsers debugger, not sure why firefox adds the px to it, super annoying.

**Comment by alicoding** [comment]
r+ with travis green :) -- Don't forget to run `grunt validate` please.

**Pull request merged**